Formula/csound.rb

```diff
@@ -75,19 +84,23 @@ class Csound < Formula
   end
 
   def install
-    ENV["JAVA_HOME"] = Formula["openjdk"].libexec/"openjdk.jdk/Contents/Home"
-
-    system "cmake", "-S", ".", "-B", "build",
-                    "-DBUILD_JAVA_INTERFACE=ON",
-                    "-DBUILD_LUA_INTERFACE=OFF",
-                    "-DCMAKE_INSTALL_RPATH=@loader_path/../Frameworks;#{rpath}",
-                    "-DCS_FRAMEWORK_DEST=#{frameworks}",
-                    "-DJAVA_MODULE_INSTALL_DIR=#{libexec}",
-                    *std_cmake_args
+    ENV["JAVA_HOME"] = Formula["openjdk"].libexec/(OS.mac? ? "openjdk.jdk/Contents/Home" : "lib")
```

Comment:
```suggestion
    ENV["JAVA_HOME"] = Language::Java.overridable_java_home_env
```
This occurs commonly enough in formulae that we have a dedicated method for it.

Reply:
Thanks, this is now done.